arangod/Transaction/Manager.cpp

```diff
@@ -312,15 +316,71 @@ Result Manager::createManagedTrx(TRI_vocbase_t& vocbase, TRI_voc_tid_t tid,
     auto& buck = _transactions[bucket];
     auto it = buck._managed.find(tid);
     if (it != buck._managed.end()) {
+      if (ServerState::instance()->isDBServer() && isFollowerTransactionId(tid)) {
```

Comment:
Do we need a case here that we are looking at a legacy transaction id and yet this is a follower transaction? What if the transaction has the hint IS_FOLLOWER_TRANSACTION?

Reply:
Done